go.mod

```diff
@@ -123,21 +124,35 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 )
 
+// the sdk's mandatory replaces
 replace (
-	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
-	github.com/aws/aws-sdk-go v1.25.48 => github.com/aws/aws-sdk-go v1.33.0
-	github.com/aws/aws-sdk-go v1.27.0 => github.com/aws/aws-sdk-go v1.33.0
-	github.com/cosmos/cosmos-sdk => github.com/classic-terra/cosmos-sdk v0.45.13-classic
-	github.com/cosmos/ledger-cosmos-go => github.com/terra-money/ledger-terra-go v0.11.2
-	github.com/docker/docker v1.4.2-0.20180625184442-8e610b2b55bf => github.com/docker/docker v1.6.1
-	github.com/ethereum/go-ethereum v1.9.25 => github.com/ethereum/go-ethereum v1.10.9
-	github.com/gin-gonic/gin v1.4.0 => github.com/gin-gonic/gin v1.7.7
+	// use cosmos fork of keyring
+	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	// dgrijalva/jwt-go is deprecated and doesn't receive security updates.
+	// TODO: remove it: https://github.com/cosmos/cosmos-sdk/issues/13134
+	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.4.2
+
+	// Fix upstream GHSA-h395-qcrw-5vmq vulnerability.
+	// TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
+	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.8.1
```

Comment:
it matters

Reply:
If you read the actual issue they are stating the following: "Sure! Make sure to double check that all cosmos deps that import gin-gonic/gin, actually already updated their gin-gonic/gin dep to >=1.7.0."

We already have a redirect targeting v1.7.7 for packages using 1.4.0. However do we know that all indirect dependencies will be OK with a bump to v1.8.1? For all we know some of our indirect dependencies like Rosetta, e.g, might be asking for 0.x.y which given your new replace rule would lead to breaking changes as they resolve incompatible major versions of a package that might or might not be backwards compatible due to the fact that SemVer major version bumps do not offer any guarantees in that regard :shrug: